features/step_definitions/assertion_steps.rb

```diff
@@ -1,3 +1,8 @@
 Then("I should see {string}") do |text|
   expect(page).to have_content text
-end
\ No newline at end of file
+end
+
+Then("I should be on {string} category page") do |category|
+  visit (`/categories/${category[id]}`)
```

Comment:
is this the best way of writing this step definition?

Reply:
We realized now that the local variable |category| is not referring to the category[id], meaning it is irrelevant for the test.. BUT since it is expecting a `{string}` expression in this test, we need to refer to something. So we cannot think of a better solution at this moment.. What is your suggestion?